### DISCUSSION TITLE
How map a mouse button click to open a new tab in Chrome, etc.`

### DISCUSSION THREAD

**Discussion by leotong** [noah-nuebling/mac-mouse-fix#929] [Questions]
Is there a way I can map a mouse button click to open a new tab in Chrome or other browsers?

e.g. by mapping ⌘ + [mouse click] or some other way of mapping "Open link in new tab" to a mouse button

Thanks!

**Comment by ssw112** [comment]
I ended up using keyboard maestro to get this shortcut to work
